kfs-core/src/main/java/edu/arizona/kfs/sys/KFSConstants.java

```diff
@@ -6,6 +6,7 @@
     public static final String DUPLICATE_INVOICE_QUESTION_ID = "DVDuplicateInvoice";
 
     public static final String GL_ENTRY_IMPORTING = "glEntryImporting";
+    public static final String DOC_FORM_KEY_88888888 = "88888888";

```

Comment:
It's what's in the URL for all Documents. No idea what it means, but it's needed. I've tried searching for where it's used, with little success on finding an explanation.